### DISCUSSION TITLE
Documentation for Reports & Dashboards

### DISCUSSION THREAD

**Discussion by salonichaudhry** [Sunbird-Obsrv/Community#3] [Documentation]
Hi,

As a part of UCI, we are enabling the creation of aggregate Superset dashboards and also data exhaust report downloads via the Console. I was wondering if there is any documentation that exists for either of these?

Thanks!

**Comment by rhwarrier** [comment]
We have a doc on [Data Exhaust APIs](https://docs.google.com/document/d/1cls5ak-ZFcV11W4rcJl01zmlXIZMSXIgmXH1DF0iDDM/edit#heading=h.nj23sjpj5u97) as well as a [Report Configurator guide](https://docs.google.com/document/d/1SMtR0gh66lLE7FUPC9km9RwXMoyHWajWp-ToH8BvtTE/edit) 
You can access these via the docs now - we'll bring them into appropriate sections on the Obsrv microsite soon enough.
Thanks

**Comment by salonichaudhry** [comment]
Hi @rhwarrier, have requested access to the 'Report Configurator guide'.

**Reply by pramodkvarma** [reply]
Isn't it a public doc @rhwarrier ? Can you bring it onto obsrv site? You can easily do it through Gitbook yourself. Take help of @surendrasinghs

**Reply by aayushkedia95** [reply]
@rhwarrier, can you provide access to the 'Report Configurator Guide'?

**Reply by pramodkvarma** [reply]
Where is this doc? Can we move it here without needing any permissions? @rhwarrier ?

**Reply by rhwarrier** [reply]
I'd shared the doc with Saloni the same day. Will get the doc to the microsite so that it can be accessed without any such dependency

**Comment by aayushkedia95** [comment]
@rhwarrier, we've gone through the shared documentation. However, based on our conversations with Sai, the process seems to include the creation of a dataset, and relevant jobs for dataset update. This is however missing in the current documentation. Can somebody from the team help us here, please? 

cc - @ChakshuGautam

**Reply by rhwarrier** [reply]
@anandp504 can you please help with relevant documentation that's available?

**Reply by anandp504** [reply]
@aayushkedia95 Please find the API documentation for the data exhaust apis [here](http://docs.sunbird.org/latest/apis/dataexhaustapi/index.html).

**Reply by pramodkvarma** [reply]
Can we ensure Obsrv documentation reflect these instead of people having to search discussion board? Thanks @anandp504
